plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -371,11 +367,4 @@ private ResourceGroup getRootGroup(ResourceGroupId groupId)
         // GroupId is guaranteed to be in groups: it is added before the first call to this method in configure()
         return groups.get(groupId);
     }
-
-    @Managed
```

Comment:
Why? Please add the reason to commit body.

Reply:
- refreshFailures is not used anywhere else, so unnecessary code/imports/dependencies to update refreshFailures variable. refreshFailures only increases, never resets as of now (no need for it)
- Currently, `lastRefresh.set(0);` represents that config is not loaded. Only after lastRefresh time is set in Line 227 `lastRefresh.set(System.nanoTime())`, getSelectors() function or match() function in line 164, 147 in respectively does not throw error and fetches config data. 
- configExecutor scheduler must continue to try to read config forever; if configExecutor stops, server needs to restart for scheduler to start again
-  So, there is no need to count how many refresh failures happened because scheduler would never stop refreshing.

Reply:
> refreshFailures is not used anywhere else, so unnecessary code/imports/dependencies 

Please consider JMX metrics.

Reply:
oh JMX ! Undo delete.